terraform-modules/aws/msk/main.tf

```diff
@@ -0,0 +1,164 @@
+locals{
+  years_valid = 10 
```

Comment:
Probably should just make this an input var  and let the user decide what they want to set this to =)

Reply:
done

Reply:
variable "years_valid" {
  type        = number
  default     = 10
  description = "The Certificate will be valid for a period of x years from the time it is issued."
}

Reply:
declarations
https://github.com/ManagedKube/kubernetes-ops/pull/418/files#diff-7cf35ca51c3d00cb2975cb66db673cb0873ed2f4a3fe6c60bfaafc5647ae2930R167-R171

Reply:
use : https://github.com/ManagedKube/kubernetes-ops/pull/418/files#diff-39fce5dc1d6ea7e5d59552eb1bb9a608452d260fc286d1963344fc0407e8c5d0R1-R3